tests/HUBActionRegistryTests.m

```diff
@@ -58,7 +58,7 @@ - (void)testRegisteringFactoryAndCreatingAction
     HUBActionFactoryMock * const factory = [[HUBActionFactoryMock alloc] initWithActions:@{actionIdentifier.namePart: action}];
     [self.actionRegistry registerActionFactory:factory forNamespace:actionIdentifier.namespacePart];
     
-    XCTAssertEqual([self.actionRegistry createCustomActionForIdentifier:actionIdentifier], action);
+    XCTAssertEqualObjects([self.actionRegistry createCustomActionForIdentifier:actionIdentifier], action);
```

Comment:
Here we actually want to make sure that the objects are the same **instance** so `XCTAssertEqual` should be used.